src/services_models/services_constructor.jl

```diff
@@ -169,3 +182,50 @@ function construct_service!(
     smooth_ace_pid!(psi_container, services)
     aux_constraints!(psi_container, sys)
 end
+
+"""
+    Constructs a service for StaticReserveGroup.
+"""
+function construct_service!(
+    psi_container::PSIContainer,
+    services::Vector{SR},
+    ::PSY.System,
+    model::ServiceModel{SR, GroupReserve},
+    ::Dict{Symbol, DeviceModel},
+    ::Vector{<:DataType},
+) where {SR <: PSY.StaticReserveGroup}
+    time_steps = model_time_steps(psi_container)
+    names = (PSY.get_name(s) for s in services)
+
+    if model_has_parameters(psi_container)
+        container = add_param_container!(
+            psi_container,
+            UpdateRef{SR}("service_requirement", "get_requirement"),
+            names,
+            time_steps,
+        )
+        get_parameter_array(container)
```

Comment:
what does this line do? it looks like it returns an array that's not assigned/used anywhere?

Reply:
Not sure haha I copied from the `construct_service` for `RangeReserve`. @daniel-thom, can you help us here?